### PR TITLE
feat:added all screens of pomo app

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
added all pomo screens.

### CODE REVIEW

**Comment by TusharJade** [comment]
Everything looks good, the code is clean app is also responsive. Good to go for merge 👍

**Review by TusharJade — APPROVED** [review]
Everything looks good, the code is clean app is also responsive. Good to go for merge 👍

File: src/Screens/PomodoroPage/pomoPage.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    background: linear-gradient(to right, #16222a, #3a6073);
+    object-fit: cover;
+    border-image-width: 20px;
+    padding: 10px;
+    margin-top: 10px;
+    border-radius: 10px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
```
Comment (MaruthiPothuganti) [inline src/Screens/PomodoroPage/pomoPage.css]: https://github.com/SumitSaundadkar/mypomodoroapp/pull/1/files#:~:text=.pomo_watch_container%7B-,display%3A%20flex%3B,align%2Ditems%3A%20center%3B,-background%3A%20linear


This already makes the container center. So, there is no use of the above lines

**Review by MaruthiPothuganti — COMMENTED** [review]
Code looks clean and UI looks good. Suggested a change. Everything else looks good to merge.

File: src/Screens/PomodoroPage/pomoPage.css (hunk 1 above)
Comment (SumitSaundadkar) [reply src/Screens/PomodoroPage/pomoPage.css]: done

**Pull request merged**